spec/services/filter_application_choices_for_providers_spec.rb

```diff
@@ -26,6 +26,12 @@
       expect(result).to eq([application_choices.last])
     end
 
+    it 'filters by candidate name with superfluous spaces' do
+      result = described_class.call(application_choices: application_choices, filters: { candidate_name: " #{application_choices.last.application_form.last_name} " })
+
+      expect(result).to eq([application_choices.last])
```

Comment:
❄️

Reply:
this seems to have caused a flakey test due to using `last`, could we assign the application to a variable instead?